server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+
+// TODO - Change this to use environment variables
+const port = 8081;
+
+const eventType = [ 
+    {label : "cop"},
+    {label : "landl"},
+    {label : "learningEvents"} 
+];
+
+// Dynamic endpoint receiving a param of eventtype to test
+app.get('/events/:eventtype', function (req, res) {
+    let foundEvent = eventType.find((event) => {
```

Comment:
`foundEvent ` should be a `const` since is not modified anywere

Reply:
Will do :)